airbyte-webapp/src/components/ui/Switch/Switch.module.scss

```diff
@@ -41,7 +42,7 @@
 
     &::before {
       position: absolute;
-      z-index: 1;
+      z-index: z-indices.$switchSliderBefore;
```

Comment:
Moving this to a variable.  It was causing a bug where users could interact with the switches behind the backdrop.  We really want any z-index in the global file to prevent collisions.